base_miner/UCF/README.md

```diff
@@ -0,0 +1,12 @@
+## UCF
+
+This model has been adapted from [DeepfakeBench](https://github.com/SCLBD/DeepfakeBench).
+
+## Setup Instructions
+
+1. **Download Model Weights**:
+   - Download the `ucf_best.pth` and `xception_best.pth` files from the [DeepfakeBench Releases](https://github.com/SCLBD/DeepfakeBench/releases/tag/v1.0.1) page.
+
+2. **Prepare the Weights Directory**:
+   - Create a directory named `weights/` in your project root.
+   - Place the downloaded files into this `weights/` directory.
```

Comment:
[Suggestion] This happens automatically now right? If so, lets drop the setup instructions